compiler/luci/pass/include/luci/Pass/QuantizeDequantizeWeightsPass.h

```diff
@@ -31,23 +31,38 @@ namespace luci
  */
 class QuantizeDequantizeWeightsPass : public logo::Pass
 {
+public:
+  struct Context
+  {
+    loco::DataType input_model_dtype = loco::DataType::Unknown;
+    loco::DataType output_model_dtype = loco::DataType::Unknown;
+    QuantizationGranularity granularity = QuantizationGranularity::ChannelWise;
```

Comment:
Can we create `QuantizationGranularity::Unknown` and initialized `granularity` with `unknown` just like `input_model_dtype` and `output_model_dtype`? 
Initializing to channel by dfault is good to some NPU but may not be good to some other mechanisms which ONE needs to support (in future, which I don't know now 😸).

Reply:
I totally agree with you. I've posted  #8472 for this issue :)